test/e2e/utils/utils.go

```diff
@@ -275,11 +275,13 @@ func ValidateLogicalLinkIsDisk(instance *remote.InstanceInfo, link, diskName str
 
 	devFsPath, err := instance.SSH("find", link, "-printf", "'%l'")
 	if err != nil {
-		return false, fmt.Errorf("failed to find symbolic link for %s. Output: %v, errror: %v", link, devFsPath, err.Error())
```

Comment:
Why do we skip instead of erroring here?

Reply:
This function is called in a loop, based on pre-defined device paths. These device paths are generated here: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/e26e399e3d623b6c6b339201a1860159fcb9fbc7/pkg/deviceutils/device-utils.go#L116

If a device path that was generated isn't actually valid (eg: find fails to get a link), we should skip over the path, and move on to the next. So in that way, we don't actually want to fail here, we just want to return false to the caller, to indicate that the provided "link" is not a disk.